### PR TITLE
1: Feature: Add summary indicators to activity section.

### PR DESCRIPTION
**Pull request by knowuh**
This adds summary indicators to the activity feedback section.
Also adds a summary indicator to the rubric-test page.

PT Story:
[#155404659]

Add summary feedback indicators to the summary portal reports.
For numerical feedback, post the mean.
For rubric feedback, generate an icon that uses proportional color bars to represent what percentage of students scored at a particular level for each category.

https://www.pivotaltracker.com/story/show/155404659

### CODE REVIEW

File: js/containers/rubric-test.js [hunk 1]
```diff
@@ -48,7 +48,23 @@ const sampleRubric = {
...
+  const answers = []
 
+  for (let i = 0; i < numAnswers; i++) {
+    const answer = {}
+    rubric.criteria.forEach(c => {
+      answer[c.id] = pickRating()
+    })
+    answers.push(answer)
+  }
+  console.dir(answers)
```
Comment (pjanik) [inline js/containers/rubric-test.js]: Perhaps it shouldn't be there?

File: js/components/summary-indicator.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+        {rows.map((r) => this.renderRubricRow(r))}
+      </div>
+    )
+  }
+
+  render () {
+    const {scores, useRubric, showScore, rubricFeedbacks} = this.props
+    const _showScore = showScore && scores && scores.length > 0
+    const showRubric = useRubric && rubricFeedbacks && rubricFeedbacks.length > 0
+    const showLabel = showRubric || showScore
```
Comment (pjanik) [inline js/components/summary-indicator.js]: `|| showScore` or `|| _showScore`? 😉

Comment (knowuh) [reply js/components/summary-indicator.js]: Ah! Nice catch. Thanks

File: js/containers/rubric-test.js (hunk 1 above)
Comment (knowuh) [reply js/containers/rubric-test.js]: Thanks --  you are right.

**Pull request merged**